app/src/test/java/com/kickstarter/viewmodels/RewardViewModelTest.java

```diff
@@ -497,6 +497,28 @@ public void testUsdConversionNotShownForUSProject() {
   }
 
   @Test
+  public void testUsdConversionTextRoundsUp() {
+    // Set user's country to US.
+    final Config config = ConfigFactory.configForUSUser();
+    final Environment environment = environment();
+    environment.currentConfig().config(config);
+    final RewardViewModel vm = new RewardViewModel(environment);
+
+    // Set project's country to CA and reward minimum to $0.30.
+    final Project project = ProjectFactory.caProject();
+    final Reward reward = RewardFactory.reward().toBuilder().minimum(0.3f).build();
+
+    final TestSubscriber<String> usdConversionTextViewText = TestSubscriber.create();
+    vm.outputs.usdConversionTextViewText().subscribe(usdConversionTextViewText);
+    final TestSubscriber<Boolean> usdConversionSectionIsHidden = TestSubscriber.create();
```

Comment:
We can remove this and the next line since we don't assert on the `usdConversionSectionIsHidden` subscription, otherwise happy for this to be merged